docs/exploration-and-organization/events-and-timelines.md

```diff
@@ -12,19 +12,21 @@ Events and timelines are a way to capture that chronological knowledge and make
 
 ## Events
 
+![An event on a time series](./images/event-on-a-time-series.png)
+
 An event is basically a date + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
 
```

Comment:
I suggest adding: 
- Where the events "live" (on collection level if I understand it correctly -- and they don't apply to subcollections)
- Where can they be displayed (on questions, i think?)

This is based on my own experience. I remember when I was reading this article for onboarding, it was really difficult for me to create a mental model of events relating to other entities in MB like dashboards, questions, etc. 
Also surprising was to find out that events of top collections are not visible from subcollections -- considering you can apply them temporarily